### PR TITLE
display totals & exchange rate in usd

### PR DESCRIPTION
**Pull request by whilei**
Currently hardcoded as USD, but opens the door to localizing currency rates and displays. 

I think there is an issue with `toLocaleString()` not working as expected... maybe because of webpacking?

- 0e02b9b Formats header to look more like latest mockup design

### CODE REVIEW

File: src/store/accountActions.js [hunk 1]
```diff
@@ -132,12 +133,14 @@ export function importWallet(wallet, name, description) {
                         type: 'ACCOUNT/IMPORT_WALLET',
                         accountId: result,
                     });
-                    dispatch(loadAccountBalance(result));
-                    if (address(result))
+                    // We should load balance and add the new account to the canonical list.
+                    if (address(result)) {
+                        dispatch(addAddress(result, name, description));
```
Comment (elaineo) [inline src/store/accountActions.js]: `addAddress` puts the address into the address book as well as the account list

File: src/components/layout/total.js [hunk 2]
```diff
@@ -22,24 +29,89 @@ const Render = ({ total }) => {
...
+                cny: rates.get('cny'),
+            };
+            fiat = {
+                total: {
+                    btc: total.getFiat(r.btc),
+                    eur: total.getFiat(r.eur),
+                    usd: total.getFiat(r.usd),
+                    cny: total.getFiat(r.cny),
+                },
+                pair: {
```
Comment (elaineo) [inline src/components/layout/total.js]: I don't understand the purpose of `pair`

Comment (whilei) [inline src/components/layout/total.js]: Thank you -- it was redundant to `r`; my reasoning was I wasn't totally our API exchange rates were going to be in Ether units (vs mwei, wei...) and thus had left the onus to `getFiat` to handle it. Have removed it.

File: src/store/accountActions.js (hunk 1 above)
Comment (whilei) [inline src/store/accountActions.js]: I changed it `loadAccountsList()` which seems like overkill... is there another method you would suggest for pushing the new account just to the list of controlled accounts?

Comment (elaineo) [inline src/store/accountActions.js]: actually there should be an  `ADD_ACCOUNT` dispatch like in `createAccount` but i see we forgot to add it. the reducer will then add it to `accountsList`

```Javascript
dispatch({
                type: 'ACCOUNT/ADD_ACCOUNT',
                accountId: result,
                name,
                description,
            });
```

**Comment by whilei** [comment]
Don't know why Travis no like... ?

**Pull request merged**